sdks/python/apache_beam/runners/portability/fn_api_runner/translations_test.py

```diff
@@ -254,7 +254,6 @@ def expand(self, pcoll):
           | Create([('a', x) for x in vals])
           | 'multiple-combines' >> MultipleCombines())
 
-  @attr('ValidatesRunner')
```

Comment:
Does this work on other runners, should we just skip it for DataflowRunner?

Reply:
ping @chamikaramj I was wondering the same thing.

Reply:
I think Combiner packing was disabled for both DataflowRunner and PortableRunner so don't think this test is useful currently. We should add this back when CombinerPacking is enabled for various runners.